pkg/storage/v2/mysql/query.go

```diff
@@ -227,7 +227,7 @@ func ConstructWhereSQLString(wps []WherePart) (sql string, args []interface{}) {
 		return "", nil
 	}
 	var sb strings.Builder
-	sb.WriteString("WHERE ")
+	sb.WriteString(" WHERE ")
```

Comment:
Can you describe the change here? This will impact all the current queries.

Reply:
I am reversing it in this commit.
https://github.com/bucketeer-io/bucketeer/commit/7b0c92a29d0c06b5a9814c571d1386f4dcbfc555